### PR TITLE
Adding the Signup Element

### PR DESCRIPTION
**Pull request by zoltot**
A couple of questions, are we using percentages or rem, and do we have a naming convention for our css and javascript varaibles, and how do we want to integrate testing.

### CODE REVIEW

File: src/Components/Signup-form/SignupForm.module.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+}
+.Signup__Container{
+    background-color: #f3f3f3;
+    width: 50%;
+    min-width: 630px;
+    /* height: 44%; */
+    color: #3C5A92;
+    box-sizing: border-box;
+    font-size: rem;
```
Comment (2940cristian) [inline src/Components/Signup-form/SignupForm.module.css]: Seems the font-size was left empty here

File: src/Components/Signup-form/Signup_form.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    <form className={classes.Signup__Form}>
+
+    <div className="Register Schoool">
+
+    <div className={classes.Header}>
+    <h1>Register School</h1>
+    </div>
+
+    <div className={classes.MarginBellow}>
+    <label for="text"><b>School Name</b></label>
```
Comment (2940cristian) [inline src/Components/Signup-form/Signup_form.js]: I'm noticing the labels use *for* but they should be specific to each input and each input should have it's own id. Here's an example: https://www.w3schools.com/tags/tag_label.asp

**Comment by 2940cristian** [comment]
> A couple of questions, are we using percentages or rem, and do we have a naming convention for our css and javascript varaibles, and how do we want to integrate testing.

Personally I use all of these: rem, vh, and percentages. Each has it's up's and down's. 

The naming convention for css we decided to use was BEM: http://getbem.com/introduction/

Javascipt variables i'm not sure, do you have any suggestions. But make sure to follow best practices and that variables, classes, or function are descriptive. I like to use camelCase, example: const thisIsCamelCase; Where the first letter is lowercase followed by capitalized first letter words after

Testing we won't have much to test right now since it's all UI work, once we work on integrating the backend more logic should be involved we can test. Otherwise we may use enzyme to tests our styling, buttons, flow, etc.

File: src/Components/Signup-form/Signup_form.js (hunk 2 above)
Comment (zoltot) [reply src/Components/Signup-form/Signup_form.js]: THe changes were made

**Pull request merged**